infra_event_notifier/cli/terragrunt.py

```diff
@@ -28,42 +22,29 @@
 TERRAGRUNT_EVENT_SOURCE = "terragrunt"
 
 
-class SentryKubeConfig:
+class RegionsConfig:
     """
-    In order to properly log Terragrunt events, we need some config from
-    sentry-kube. But we don't want to install sentry-kube and its 1 million
-    dependencies. This represents a simplified model of that config. For full
-    details, see repo getsentry/sentry-infra-tools path
-    libsentrykube/config.py.
+    In order to properly log Terragrunt events, we need to map the region from
+    the name of the slice to a Sentry region name. We use a config file sotred
+    in ops/ for this.
     """
 
     def __init__(self) -> None:
-        config_file_name = os.getenv(
-            "SENTRY_KUBE_CONFIG_FILE", "cli_config/configuration.yaml"
-        )
+        config_file_name = os.getenv("SENTRY_TERRAGRUNT_REGIONS_CONFIG")
+        if config_file_name is None:
+            raise ValueError(
+                "Can't locate sentry-kube config file. Please set "
+                "SENTRY_TERRAGRUNT_REGIONS_CONFIG."
+            )
```

Comment:
Would we also want an option to pass the filename as an argument?

Reply:
Yes there is no reasons to pass these as env variables. 
We use them for secrets and that's expected, we should use CLI arguments for the rest.

sentry-kube passes a lot as environment variables that should be cli arguments. I think it was done out primarily to cut  corners and have CLI argument available as global variables rather than passing them properly between modules. I would not follow that pattern.